config/locales/en.yml

```diff
@@ -2337,6 +2338,13 @@ en:
           helper_heading: How do I find my school district?
           school_district_name: School District Name
           title: Select the school district where you lived on December 31, %{filing_year}
+      ny_third_party_designee:
+        edit:
+          reveal:
+            body_html: "By selecting a third-party designee, you are authorizing NY Department of Taxation and Finance to discuss with the designee any questions related to this return. You are also authorizing the designee to give and receive confidential taxpayer information relating to: \n<br> \n<b><ul class=\"list--bulleted\"> \n  <li>this return, including missing information \n  <li>any notices or bills arising from this filing that you share with the designee (they will not be sent to the designee) \n  <li>any payments and collection activity arising from this filing, and \n  <li>the status of your return or refund. \n</ul></b>'\n"
```

Comment:
There's a trailing apostrophe here, and I think the newlines are probably being ignored and could be removed entirely.
![image](https://github.com/codeforamerica/vita-min/assets/122047394/4340390e-ff9e-407c-8e94-d9ba856f53e9)

Reply:
yeah super weird that i18n normalize added all those new lines